test/integration/storageversion/storage_version_filter_test.go

```diff
@@ -148,7 +148,9 @@ func testBuiltinResourceRead(t *testing.T, cfg *rest.Config, shouldBlock bool) {
 func TestStorageVersionBootstrap(t *testing.T) {
 	// Start server and create CRD
 	etcdConfig := framework.SharedEtcd()
-	server := kubeapiservertesting.StartTestServerOrDie(t, nil, framework.DefaultTestServerFlags(), etcdConfig)
+	flags := framework.DefaultTestServerFlags()
+	flags = append(flags, "--runtime-config=api/all=true")
+	server := kubeapiservertesting.StartTestServerOrDie(t, nil, flags, etcdConfig)
```

Comment:
This matches another call below:

https://github.com/kubernetes/kubernetes/blob/6025e23ab691286100ef6bb03c5c6a50461e88b7/test/integration/storageversion/storage_version_filter_test.go#L176-L178

I'm not sure why specific types need to be enabled explicitly when there's also an `api/all=true`.

I tried without that here, but then the test failed.